providers/go-feature-flag-in-process/go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/open-feature/go-sdk-contrib/providers/go-feature-flag-in-process
+
+go 1.22.5
```

Comment:
of course it's up do you... but consider the go SDK still only requires 1.21, so you may be excluding some adoption with this.

Reply:
Yes I know, but the library is in 1.22.5 so I have no real choice here 😒